### ISSUE TITLE
Why the cover files cannot generated through the script?

### ISSUE THREAD

**Issue by liaohu1231** [simroux/ClusterGenomes#1]

**Comment by simroux** [comment]
The cover file is generated when the script runs successfully. Do you have any error message ?

**Comment by liaohu1231** [comment]
The script runs successfully when deleted the line 103-106, I don't know why you write the three line codes.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "simroux/ClusterGenomes"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年9月9日(星期三) 晚上11:53
收件人:&nbsp;"simroux/ClusterGenomes"<ClusterGenomes@noreply.github.com&gt;;
抄送:&nbsp;"1043386801"<1043386801@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [simroux/ClusterGenomes] Why the cover files cannot generated through the script? (#1)





 
The cover file is generated when the script runs successfully. Do you have any error message ?

**Comment by simroux** [comment]
If this is v5.1, lines 103-106 are here to make sure that the mummer result file used as input is correctly sorted. So while the script may finish, you probably will want to double-check the results, because they may very well be incorrect if it only completes after removing l. 103-106.

**Comment by liaohu1231** [comment]
However，the script was geneally locked in this step if haven't deleted the three line codes, why is this happening.

**Comment by liaohu1231** [comment]
Yes, it's v5.1. However，the script was geneally locked in this step if haven't deleted the three line codes, why is this happening.
but we found the results look right after check and have a good effect for cluster of phage genome.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "simroux/ClusterGenomes"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年9月10日(星期四) 凌晨0:16
收件人:&nbsp;"simroux/ClusterGenomes"<ClusterGenomes@noreply.github.com&gt;;
抄送:&nbsp;"1043386801"<1043386801@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [simroux/ClusterGenomes] Why the cover files cannot generated through the script? (#1)





 
If this is v5.1, lines 103-106 are here to make sure that the mummer result file used as input is correctly sorted. So while the script may finish, you probably will want to double-check the results, because they may very well be incorrect if it only completes after removing l. 103-106.

**Comment by simroux** [comment]
Without seeing the files, I can't say much more than: l. 103-106 are here for a reason, so you should use a version with these lines deleted with caution and knowing that there is a risk of incorrect results.

**Comment by simroux** [comment]
Most likely what happened was the "sort" function (l. 236-243) did not work as intended. You may want to look at the files XX-nucmer.out.coords and XX-nucmer.out.coords.sorted to check the difference, and see why the latter one (XXX-sorted) is not sorted according to the first column (query sequence).

**Comment by liaohu1231** [comment]
Thank you very much，I found it can run successful if only deleted the line 106 <STDIN&gt; just now, so it will generate right results if only deleted line 106.
I'm very appreciated for your answer.



---Original---
